code/_globalvars/lists/ores_spawned.dm

```diff
@@ -0,0 +1,46 @@
+/**
+ * Sets of global lists breaking down the base spawning distributions for various maps and stations.
+ *
+ * Currently used for ore vents on roundstart when the map is generated. (See ore_vent.dm, seedRuins() and ore_generation.dm)
+ * Ore vent lists here are copied to ore_vent_minerals on ruin seeding,
+ * in order to dynamically adjust the spawn rates as materials are picked and set a global ore distribution from vents.
+ *
+ * By default vents pull 4 unique materials each, but this can vary with subtypes.
+ */
+
+GLOBAL_LIST_INIT(ore_vent_minerals_lavaland, list(
```

Comment:
Creating defines/lists attached to a specific map doesn't feel right. You should instead create a config file that is attached to a map and load these values from there.

Because if we rename/delete/add new maps we shouldn't have to change the code each time that happens but rather have a config file with a generic format that can be shared across all maps.

Idk just something to think about that's all

Reply:
it's less maps and more "types of mining", sorta maybe? idk. so many of these things are already built in cause they sorta they have to be.